src/scenes/Game.ts

```diff
@@ -139,8 +139,8 @@ export class Game extends Scene
         }
 
         // call update of each object
-        // Only update game objects if the game is not over
-        if (!this.isGameOver) {
+        // Only update game objects if the game is not over and not paused
+        if (!this.isGameOver && !this.isGamePaused) {
             // call update of each object
             this.gameObjects.forEach((object) => object.update());
```

Comment:
I wonder if you add the game objects to the scene, if you still need to call update manually
~
if you do this.physics.add.existing(gameObject, bodyType);

if you https://docs.phaser.io/phaser/concepts/physics/arcade#arcade-body
~

nm this is our custom objects. Maybe those game objects should call this.physics.add.existing(gameObject, bodyType); and check if game.physics is paused?